pylint_odoo/checkers/modules_odoo.py

```diff
@@ -826,13 +827,14 @@ def _check_xml_attribute_translatable(self):
         """The xml attribute is missing the translation="off" tag
             Example  <attribute name="groups">sale.group</attribute>
         """
-        self.msg_args = []
-        for xml_file in self.filter_files_ext('xml', relpath=True):
-            for record in self.get_xml_records(
-                    os.path.join(self.module_path, xml_file), None,
-                    '//attribute[not(@translation)]'):
-                self.msg_args.append(
-                    ("%s:%d" % (xml_file, record.sourceline), 'xml_id'))
-        if self.msg_args:
-            return False
+        if ODOO_VERSION == '8.0':
```

Comment:
Currently we are using [`valid_odoo_version` from MQT](https://github.com/Vauxoo/maintainer-quality-tools/blob/233d67accf3da40f628ab64c49164ff6d9c9c6cb/travis/test_pylint#L123)

We could take advantage of this variable if the value assigned is `['8.0']`
I mean,
`if self.config.valid_odoo_versions != ['8.0']: return True`

Add a tests to consider this option too